jaeseon/db/migrations/20221103074949_create_users_signup_table.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE users_signup (
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+email_id INT NOT NULL,
+password_id INT NOT NULL
+CONSTRAINT user_signup_email_id_fkey FOREIGN KEY (email_id) REFERENCES users_email(id),
+CONSTRAINT user_signup_password_id_fkey FOREIGN KEY (password_id) REFERENCES users_password(id)
+);
```

Comment:
```suggestion
CREATE TABLE users_signup (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    email_id INT NOT NULL,
    password_id INT NOT NULL
    CONSTRAINT user_signup_email_id_fkey FOREIGN KEY (email_id) REFERENCES users_email(id),
    CONSTRAINT user_signup_password_id_fkey FOREIGN KEY (password_id) REFERENCES users_password(id)
);
```
가독성을 위해 indentation을 수정해주세요~ 다른 migration 파일들도 동일하게 적용해주세요 :)

Reply:
넵, 앞공간 .... 4칸을 TAP 사용하여 줄 바꿈 수정했습니다.